templates/includes/head.jade

```diff
@@ -51,6 +51,9 @@ head
   //- js modernizr
   script(src=contents.js.vendor['modernizr-2.8.3.min.js'].url)
 
+  //- js lazysizes
+  script(src=contents.js.vendor['lazysizes.min.js'].url)
```

Comment:
Can be placed at the end of the body or at the end of the head

> This smaller script, which should include lazySizes (and all its plugins), should than be placed before any other blocking elements (i.e.: script(s)) at the end of the body or after any blocking elements (i.e.: scripts, stylesheets) in the head to load the crucial content as fast possible. (Note: It might make also sense to call lazySizes.init(); explicitly right after lazySizes and all its plugins are added.)